app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -401,15 +401,6 @@ private CharSequence getCreatedAtDescription(Date createdAt,
         }
     }
 
-    protected void setIsReply(boolean isReply) {
-        if (isReply) {
-            replyButton.setImageResource(R.drawable.ic_reply_all_24dp);
-        } else {
-            replyButton.setImageResource(R.drawable.ic_reply_24dp);
-        }
-
-    }
-
```

Comment:
Why remove this?

Reply:
It is odd having this information hidden in the reply button. And that is the only usage of that function.

But also see here: https://github.com/tuskyapp/Tusky/pull/3778#issuecomment-2092897311

Reply:
Ok, I'd also be ok with keeping the double arrow on replies because I noticed that the web interface still shows them

Reply:
Actually, when a reply is boosted, the double arrow is still useful because the "_ boosted" would hide the "replied to_" message. Sorry for the change in mind.